src/Discord/Parts/Guild/Guild.php

```diff
@@ -165,6 +166,7 @@ class Guild extends Part
         'max_video_channel_users',
         'approximate_member_count',
         'approximate_presence_count',
+        'welcome_screen',
```

Comment:
There needs to be a `getWelcomeScreenAttribute()` if the `welcome_screen` attribute is to return an instance of `WelcomeScreen`